<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="a">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="operacja">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="oblicz">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wynik">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="pokaz_autora">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaż autora</string>
    </property>
   </widget>
   <widget class="QLabel" name="autor">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>???</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="kalendarz">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>304</width>
      <height>163</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="data">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz datę</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="suwak">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>20</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_suwak">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj nazwisko :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nazwisko">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dodaj">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="notes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>161</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dane</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Podaj liczbę :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="liczba">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>liczba pozycji :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_liczbaPozycji">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wyczysc">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>wyczyść</string>
    </property>
   </widget>
   <widget class="QLabel" name="suma">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>suma : </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="dane2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="podaj_liczbe">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Podaj liczbę :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusplus">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zapis">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>zapis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="odczyt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>odczyt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusty">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>pusty</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>kolor formularza</string>
    </property>
    <widget class="QRadioButton" name="czerwony">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>czerwony</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="zielony">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>zielony</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="szary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>szary</string>
     </property>
    </widget>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>a</zorder>
   <zorder>operacja</zorder>
   <zorder>b</zorder>
   <zorder>oblicz</zorder>
   <zorder>wynik</zorder>
   <zorder>pokaz_autora</zorder>
   <zorder>autor</zorder>
   <zorder>kalendarz</zorder>
   <zorder>data</zorder>
   <zorder>suwak</zorder>
   <zorder>label_suwak</zorder>
   <zorder>label</zorder>
   <zorder>nazwisko</zorder>
   <zorder>label_2</zorder>
   <zorder>dodaj</zorder>
   <zorder>notes</zorder>
   <zorder>lista</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>liczba</zorder>
   <zorder>label_5</zorder>
   <zorder>label_liczbaPozycji</zorder>
   <zorder>wyczysc</zorder>
   <zorder>suma</zorder>
   <zorder>label_6</zorder>
   <zorder>dane2</zorder>
   <zorder>podaj_liczbe</zorder>
   <zorder>plusplus</zorder>
   <zorder>zapis</zorder>
   <zorder>odczyt</zorder>
   <zorder>pusty</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1299</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Kalkulator">
    <property name="title">
     <string>Kalkulator</string>
    </property>
    <addaction name="pokaz"/>
    <addaction name="ukryj"/>
   </widget>
   <widget class="QMenu" name="menuEdycja_notesu">
    <property name="title">
     <string>Edycja notesu</string>
    </property>
    <addaction name="actionKopiuj"/>
    <addaction name="actionWytnij"/>
    <addaction name="actionWklej"/>
   </widget>
   <widget class="QMenu" name="menuKomunikat">
    <property name="title">
     <string>Komunikat</string>
    </property>
    <addaction name="actionk1"/>
    <addaction name="actionk2"/>
   </widget>
   <addaction name="Kalkulator"/>
   <addaction name="menuEdycja_notesu"/>
   <addaction name="menuKomunikat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="pokaz">
   <property name="text">
    <string>Pokaz</string>
   </property>
  </action>
  <action name="ukryj">
   <property name="text">
    <string>Ukryj</string>
   </property>
  </action>
  <action name="actionKopiuj">
   <property name="text">
    <string>Kopiuj</string>
   </property>
  </action>
  <action name="actionWytnij">
   <property name="text">
    <string>Wytnij</string>
   </property>
  </action>
  <action name="actionWklej">
   <property name="text">
    <string>Wklej</string>
   </property>
  </action>
  <action name="actionk1">
   <property name="text">
    <string>k1</string>
   </property>
  </action>
  <action name="actionk2">
   <property name="text">
    <string>k2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
